### ISSUE TITLE
Connect Crashes on Publish when a Repeat includes a Geopoint Question

### ISSUE THREAD

**Issue by broecher** [Esri/Survey123Community#525]
When I attempt to publish most surveys, Connect says “creating feature service”, “updating feature details”, and then it crashes and closes (even after a reinstall). Further attempts give me an error, “folder not available, unable to create folder” until I delete the folder out of AGOL. The survey doesn't show up in Survey123 Hub when this happens so you can't delete it from there.

We suspected this had to do with naming problems, but after lots of testing by modifying the default template gradually, I couldn't identify any problem with my names. In fact I had a survey working, then I backed up the xls and deleted the survey from everywhere, and now it crashes trying to publish that same xls.

**Comment by kevintheflesh** [comment]
+1

**Comment by survey123** [comment]
Hi. We are sorry to hear this. We have been unable to reproduce this in our test machines, so it is important that we understand your environment.
* What is the operating system you are running?
* Can you gather any logs providing information about the error? For example, in Windows logs from the [Event Viewer](https://technet.microsoft.com/en-us/library/cc749408%28v=ws.11%29.aspx).

**Comment by broecher** [comment]
Hi, thanks for responding. Here is a Connect log from just now when it crashed while attempting to publish. It looks like all the version and OS information is in the log. I am unable to open the event viewer because it's restricted.  I am on a physical machine. 

[Survey123ConnectforArcGIS-20160722-112005.log.txt](https://github.com/Esri/Survey123Community/files/378659/Survey123ConnectforArcGIS-20160722-112005.log.txt)

**Comment by survey123** [comment]
Thanks. Can you please also send the XLSFile? I understand that this does not seem to be the issue, but will help  for completeness on our side.

**Comment by broecher** [comment]
Here it is. Nothing fancy, I'm just learning how to use Survey123 at this point.

[R3 Deer Telemetry.xlsx](https://github.com/Esri/Survey123Community/files/378699/R3.Deer.Telemetry.xlsx)

**Comment by broecher** [comment]
I can publish the samples which are relatively large surveys. It must be something with my xls file, but I don't know what.

**Comment by broecher** [comment]
I tried to publish the xlsx above from another computer with windows 10 and Survey123 1.2.62 and it succeeded. Then I uninstalled and installed 1.5.35 and publish was successful again. Then I published again and it failed in the same way it has been. It continues to fail now.

Between each publish attempt I deleted it from everywhere, including checking AGOL.

[fail Survey123ConnectforArcGIS-20160722-180442.log.txt](https://github.com/Esri/Survey123Community/files/379212/fail.Survey123ConnectforArcGIS-20160722-180442.log.txt)

[success Survey123ConnectforArcGIS-20160722-180142.log.txt](https://github.com/Esri/Survey123Community/files/379213/success.Survey123ConnectforArcGIS-20160722-180142.log.txt)

**Comment by survey123** [comment]
@broecher   Looking at your R3.Deer.Telemetry survey I found the following:
* You are adding a geopoint within a repeat block.  This is fair game, but 1.5.35 introduced a bug on Windows which was making Connect crash randomly (more often than not, actually, but somewhat randomly. We put a fix in to this in 1.6. More info about 1.6 can be found in our [GeoNet Survey123 Blog](https://geonet.esri.com/groups/survey123/blog/2016/07/26/227-pi-approximation-day-release). I suggest you download Survey123 Connect 1.6 and try again. It should work.
* We have not resolved the Folder not found errors yet. So you will  want to be careful. By now it looks  like you know what to do, but this [GeoNet Thread](https://geonet.esri.com/thread/178120) may help in case of doubt.
* I noticed you have a first question for  the name of the field user, and then a repeat.  It is of course all your call, but I suggest that you ask people to complete a new survey for every observation. In practice, we will be storing the name of the logged-in person in every row, so there is  no need to capture the name of the person (there should not be).  More info about it in this [blog post](https://geonet.esri.com/groups/survey123/blog/2015/08/24/keeping-track-of-when-where-and-by-whom-data-is-captured)
